sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_AnalyzeActions.md

```diff
@@ -117,5 +117,7 @@ To run multiple actions in multiple documents, call `StartAnalyzeActionsAsync` o
 }
 ```
 
+See the [README][README] of the Text Analytics client library for more information, including useful links and instructions.
```

Comment:
Tip: you don't need to use this syntax when the text is the same as the named link. You can just do `... [README] ...`.